substrate/Makefile

```diff
@@ -12,14 +12,8 @@ verify: codegen ## Verify code. Includes dependencies, linting, formatting, etc
 	go vet ./...
 	go fmt ./...
 	golangci-lint run
-	@git diff --quiet ||\
```

Comment:
Why are we removing this, does it not work as intended?

Reply:
> Why are we removing this, does it not work as intended?

Why do we have such a check on `git diff` in `substrate` but not in `operator`.
`make verify` is a general verification of the code, should be happening before code commit.
Additional reason, if we want to make a script with `make verify`, code commit is always a prerequisite because the `@git diff --quiet` will fail.

Reply:
maybe we can split this into different commands like
```
vendor:
     go mod tidy
     go mod download
fmt:
    go vet ./...
    go fmt ./...
verify: codegen vendor fmt
   golangci-lint run
   @git diff --quiet ....
```

The reason we want git diff is because we want to make sure in CI that there are no files someone has missed to update.